src/AWS.Deploy.CLI/ServerMode/Controllers/DeploymentController.cs

```diff
@@ -546,5 +582,62 @@ private Orchestrator CreateOrchestrator(SessionState state, IServiceProvider? se
                                     serviceProvider.GetRequiredService<IDirectoryManager>()
                                 );
         }
+
+        private async Task<Dictionary<string, string>?> GetResourcesByTypeHint(IAWSResourceQueryer awsResourceQueryer, Recommendation recommendation, OptionSettingItem optionSetting)
```

Comment:
We need to unify the CLI and ServerMode to use the same code to fetch resources. The goal should be if I want to add another simple typehint of list of AWS resources should just require an update in one area and both CLI and ServerMode get the update.

Reply:
Done